### ISSUE TITLE
Access Point on 5Ghz using wifiphisher

### ISSUE THREAD

**Issue by rafaleon69** [wifiphisher/wifiphisher#1073]
Hello, thanks for all the work that has gone into making wifiphisher. Very useful tool! I have a problem/suggestion maybe somebody can help. I am using Realtek Semiconductor Corp. RTL8812AU 802.11a/b/g/n/ac WLAN Adapter external adapter, capable of being set as an AP both of 2.4Ghz and 5Ghz. 

Tested with hostapd v.2.6, my config:

`interface=wlan1
ssid=myssid
country_code=SE
ieee80211d=1
hw_mode=a
channel=40
macaddr_acl=0
auth_algs=3
wmm_enabled=1
ieee80211ac=1
vht_capab=[SHORT-GI-80][SHORT-GI-160][VHT160-80PLUS80][MU-BEAMFORMER][MU-BEAMFORMEE][SU-BEAMFORMER][SU-BEAMFORMEE]
require_vht=0
#vht_oper_chwidth=1                                                                                                                                           
vht_oper_centr_freq_seg0_idx=40
#vht_oper_centr_freq_seg1_idx=44
`
AP is created and my android phone recognizes it as a 5GHz AP myssid. 

At the moment I am interested in how I can create an AP point like that using wifiphisher. I tried with roguehostapd and specifying option: -fH ( to use the modified one )  but channel is force into 6, and created AP remains on channel 2.4Ghz.

Question: how to achieve a 5GHz AP with wifiphisher?
I am using latest 1.4Git version. Any help will be appreciated.

**Comment by sophron** [comment]
Roguehostapd is a fork of hostapd so things shouldn't have been different.

Why do you need a 5GHz AP using Wifiphisher? What are you trying to achieve?

**Comment by genericc** [comment]
yes i have 2 same  I am using Realtek Semiconductor Corp. RTL8812AU 802.11a/b/g/n/ac WLAN Adapter external adapter, capable of being set as an AP both on 2.4Ghz and 5Ghz to and cant deauth clients from routers this needed because al new router now support 5ghz band so all phones sopport now 5 ghz and many laptops using 5 ghz so on 2.4ghz make deauth its easy.no problem but with 5ghz it is big problem for me i have work with this issue about 1 moth every day and no result.
